Added: language support (#270)
tldr-pages/tldr-node-client: README.md

- `tldr --render <path>` render a local page for testing purposes

> To view tldr pages for a different language, set an environment variable `LANG` containing a valid [BCP-47](https://tools.ietf.org/html/bcp47) code. And then run the above commands as usual. Currently, we have translations for `zh`, `pt-BR`, `it`, `ta`, `fr`. For an updated list of translations, please refer to the main [tldr](https://github.com/tldr-pages/tldr) repo.

Reviewer: Instead of phrasing this as a list of existing translations, it would be easier to just give some examples of codes that can be used. That way there won't be a need to constantly update this list as translations in more languages are contributed. For example:

```suggestion
To view tldr pages for a different language, set an environment variable `LANG` containing a valid [BCP-47](https://tools.ietf.org/html/bcp47) code (such as `zh`, `pt-BR`, or `fr`) and then run the above commands as usual. For the list of available translations, please refer to the main [tldr](https://github.com/tldr-pages/tldr) repo.
```

Author: Sure

- `tldr --render <path>` render a local page for testing purposes

> To view tldr pages for a different language, set an environment variable `LANG` containing a valid [POSIX locale](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html#Locale-Names) (such as `zh`, `pt_BR`, or `fr`) and then run the above commands as usual. In most `*nix` systems, this variable will already be set.

Reviewer: Maybe prefix this with information about what language is used by default. Something like this:

> tldr-pages defaults to showing pages in the current language of the operating system, or English if that's not available.

WDYT? (Assuming it accurately described the actual behavior, of course.)

Author: Sounds good.

For the list of available translations, please refer to the main [tldr](https://github.com/tldr-pages/tldr) repo.

> - `LANG=zh tldr <command>`

Reviewer: I think it would be better (sequence-wise) to have this example before the previous sentence.

> - `LANG=zh tldr <command>`

Reviewer 2: ```suggestion
It is suggested that the `LANG` environment variable be set system-wide if this isn't already the case. Users without `sudo` access can set it locally in their `~/.profile`.

- `LANG=zh tldr <command>`
```

#### Installation Issues

> - If you are trying to install as non-root user (`npm install -g tldr`) and get something like -

Reviewer: I know it's not related to the changes in this PR, but the appropriate punctuation here would be `:`, not `-`. Since you're already editing these lines, maybe fix that too.
